hil/ext/auth/database.py

```diff
@@ -63,9 +63,9 @@ def set_password(self, password):
 @rest_call('PUT', '/auth/basic/user/<user>', schema=Schema({
     'user': basestring,
     'password': basestring,
-    Optional('is_admin'): bool,
+    'is_admin': bool,
```

Comment:
this changes the API. somebody could still directly make the API call where we want this to be optional.

Reply:
sure, will change accordingly.